src/components/FieldLabel/index.jsx

```diff
@@ -9,22 +9,13 @@ const FieldLabel = ({
   id,
   optional,
   text,
-}) => {
-  let requireText = '';
-
-  if (optional) {
-    requireText = <span className="text-info" aria-hidden>   Optional</span>;
-  }
-
-  return (
-    <div id={id} className={className}>
-      <strong>{text}</strong>
-      {helpText && id && <FieldHelp id={`${id}-help`} tip={helpText} />}
-      {requireText}
-      <div><span className="text-muted">{extraText}</span></div>
-    </div>
-  );
-};
+}) => (
+  <div id={id} className={className}>
+    <strong>{text}</strong>{optional ? ' (optional)' : ''}
+    {helpText && id && <FieldHelp id={`${id}-help`} tip={helpText} optional={optional} />}
+    <div><span className="text-muted">{extraText}</span></div>
+  </div>
+);
```

Comment:
This is a bit hard to read. Please add appropriate spacing/line breaks to make it readable.

Reply:
i have made some edits...essentially we are shifting the position of the 'optional' keyword for optional fields